cmd/wasm-client/permissions.go

```diff
@@ -1 +1,201 @@
 package main
+
+import (
+	"github.com/lightningnetwork/lnd"
+	"gopkg.in/macaroon-bakery.v2/bakery"
+)
+
+var (
+	faradayPermissions = map[string][]bakery.Op{
```

Comment:
I still get that import error (etcd + wasm related) for faraday , loop and pool. was able to import the lnd perms though 👍